.gitignore

```diff
@@ -1,19 +1,35 @@
-﻿*.aux
-*.log
-*.out
-*.gz
-*.toc
-*.pdf
-*.bcf
-*.blg
-*.bbl
-*.run.xml
-*.fls
-*.fdb_latexmk
-*.synctex.gz
-*.synctex(busy)
-*.lof
-*.xdv
-*.alt
-*.lot
-*.acr
+﻿# ignore all
+**
+
+!.gitignore
+!.editorconfig
+
+!*.md
+!*.tex
+!*.latex
+!*.bib
+
+!.github/
+!.github/workflows
+!.github/workflows/**
+
+!.vscode/
+!.vscode/**
+
+!assets/
+!assets/img/
+!assets/img/.gitkeep
+# uncomment this to check-in pictures
+# !assets/img/*.png
+# !assets/img/*.jpg

```

Comment:
🤮

Reply:
Bessere Idee oder ganz weglassen?

Reply:
Ja puh, ich würde die immer einkommentieren, weil man ja eigentlich immer mindestens ein Bild in der Arbeit hat

Reply:
Oh... hatte erwartet du hattest eine bessere Idee was den Bilder Fehler angeht... Aber ja das klingt wirklich sinnvoller :D

Reply:
```suggestion
!assets/img/*.png
!assets/img/*.jpg

```

Reply:
i like your funny diff, magic webpage